lab-said/route/route-gellery.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
```

Comment:
Gallery is misspelled in this file. It prevented any test from running because node couldn't find the routes.